homeassistant/components/bluetooth/manager.py

```diff
@@ -209,6 +209,20 @@ async def async_setup(self) -> None:
             self._bluetooth_adapters, self.storage
         )
         self.async_setup_unavailable_tracking()
+        seen: set[str] = set()
+        for address, service_info in itertools.chain(
+            self._connectable_history.items(), self._all_history.items()
+        ):
+            if address in seen:
+                continue
+            seen.add(address)
+            for domain in self._integration_matcher.match_domains(service_info):
+                discovery_flow.async_create_flow(
```

Comment:
We need to patch out this this so we don't load lots of integrations in the tests